### DISCUSSION TITLE
Suggestion & questions on "AAX Audio Converter"

### DISCUSSION THREAD

**Discussion by BudoSnapper2017** [audiamus/AaxAudioConverter#215] [Q&A]
I've tried to ask this already on reddit, but for unknown reasons they don't let me create new posts in r/audible and I fear a general post in a non-Audible related category won't be answered by audiamus himself. Hence I repeat it here:  

First of all: "AAX Audio Converter" is a great tool of course - for some reasons but especially for the fact that you can keep all original chapters of any Audible audio book. Thanks a lot for releasing this wonderful tool. I've been seaching quite a long time for something that works like "MuConvert Audible Converter" which is the most phantastic tool available - but its price is too high or you have to buy it by subscription - which is both not very nice. Way nicer would be if "AAX Audio Converter" could soon become an equal "MuConvert Audible Converter" (still) for free. What I miss extremely in "AAX Audio Converter" is a possibility to choose/adjust the bitrate of the resulting MP3s since only 128 kbps (kilobits per second) is only the quality of a 90s music cassette - while the original AAX quality is 128 kbps as AAC = Compact Disc quality. For by far most audio books a MP3 quality of 128 kbps is really good enough since they contain only speech. But I have some audio books, which I would really love to convert too, that contain waves of sound for e.g. meditation & relaxation. For such waves of sound you have to keep original CD quality - which would be as MP3 at least 192 kbps or even better VBR parameter V2. Of course I could keep the original CD quality of the Audible AAX files by just transcoding them to AAC/m4a or m4b - but by far most (and even every single one I know) MP3 players don't understand the format m4a/m4b. So there pleeeeease has to be a possibility to choose/adjust the resulting MP3 quality in you "AAX Audio Converter". PLEASE let it happen and make this possible. In "MuConvert Audible Converter" you can choose a resulting MP3 quality of up to 320 kbps - which, of course, is absolutely exaggerated for an original AAX/AAC quality of 128 kbps (equal to 192 kbps as MP3). But however at least you CAN choose/adjust a resulting bitrate. If "AAX Audio Converter" would offer the same possibility I could after all and finally stop thinking about this too expensive or subscription tool "MuConvert Audible Converter" (which I've so far tested only for free on two freely convertible Audible audio books - and it really works as perfect as such a tool should work). 

Questions on "AAX Audio Converter": 

When I choose constant bitrate for MP3 it is stereo, not joint stereo, right? 

When I choose variable bitrate for MP3 its bitrate becomes reeeeeeally low. I really hope that at least THIS is joint stereo. Is it joint stereo? Which VBR parameter is predetermined? In my tests I've found out that the VBR quality is worse than VBR parameter V5 (which would be best in my eyes for "usual" audio books containing only speech (I always use V5.2 in EAC whenever I convert an audio book I've copied from a CD), but by far not enough for those mentioned waves of sound - for them V3 or V2 would be most suitable) but better than V6 ... in case it DOES use joint stereo ... in case of only stereo I think it is closer to something like only V8/V9 because of its very low resulting bitrates.

Thanks a lot and best wishes.

**Comment by audiamus** [comment]
If you do not set bit rates in AAX Audio Converter, it will be left to FFmpeg to select a suitable bit rate for MP3. And as far as I remember, FFmpeg selects this from bit and/or sample rate of the AAC stream. And I believe the FFmpeg developers have thought about "equivalence". Therefore I understood that setting higher values would only increase file size but not improve audio quality.

**Comment by BudoSnapper2017** [comment]
> If you do not set bit rates in AAX Audio Converter, it will be left to FFmpeg to select a suitable bit rate for MP3.

Thanks for your answer. Does this mean, I can set bitrates in AAX Audio Converter? But how? I didn't see such a possibility. Of course there is this option to reduce the bitrate to a predetermined maximum limit. But that is further quality reduction - I need quite the opposite. Yes, FFmpeg chooses exactly the same bitrate for MP3 like the the bitrate of the AAC stream is - and that's the problem. 128 kbps for MP3 is a deal worse than 128 kbps for AAC. That's why I would really appreciate a possibility to predetermine the bitrate of the resulting MP3s to at least 192 kbps (which would be about equal to 128 kbps for AAC) or even better to a VBR parameter of V3 or even V2. More than 192 kbps or V2 would really not make sense since an even higher bitrate would actually only increase file size but not quality. But only 128 kbps actually reduces quality - as well as the low bitrates of VBR also seem to do since they result, as written, in a parameter of something between V5 and V6 only (if this is Joint Stereo and not only Stereo).

Fortunately I've found a solution for my problem meanwhile: First I transcode the original AAX+ files to AAC files using your "AAX Audio Converter" - this retains the original quality of the AAX+ files and removes the DRM of my own audio books I paid for. Then I use "fre:ac - free audio converter" to recode the AAC files to MP3 files with a bitrate of 192 kbps or V2 - this retains the original quality, too (minor loss of quality due to the recoding process is normality), while making the files usable for every MP3 player. Using "AAX Audio Converter" AND "fre:ac" in succession is 100 % equal to the use of the not free "MuConvert Audible Converter". It's a solution but however it would be much better if I could do this all by using the same tool - your "AAX Audio Converter".

"fre:ac" allows to set the VBR parameter accurate to a tenth between V0.0 and V9.9 - plus I can choose the recoding quality from low/very fast (9 or 10) to very high/slow (0) and I can determine if I want Joint Stereo or "only" Stereo). Another tool I used to use for years already - "XMedia Recode" - allows almost the same (except of setting the VBR parameter accurate to a tenth), but that tool unfortunalety deletes all metadata included in the transcoded AAC files while recoding them to MP3.

**Comment by audiamus** [comment]
As I said, with no setting in AAX Audio Converter, FFmpeg decides. You have to ask the developers there (or look into the FFmpeg source code) how they come to pick the bit rate.
